dist-material/release-docs/LICENSE

```diff
@@ -391,17 +391,6 @@ CC0-1.0 licenses
 HdrHistogram 2.1.12:	https://github.com/HdrHistogram/HdrHistogram , CC0-1.0 and BSD 2-Clause
 reactive-streams 1.0.2:	https://github.com/reactive-streams/reactive-streams-jvm , CC0-1.0
 
-========================================================================
-EPL licenses
-========================================================================
-
-The following components are provided under the EPL License. See project link for details.
-The text of each license is also included at licenses/LICENSE-[project].txt.
-
-    logback 1.2.3: https://github.com/qos-ch/logback: EPL 1.0
```

Comment:
I'm afraid this needs to be reserved because we still use logback in web app and it's in our distributed tar.

Reply:
Ok, I see the web app has used `logback-classic` and `logback-core`, I will added back.